conda_forge_tick/auto_tick.xsh

```diff
@@ -17,10 +17,9 @@ from .path_lengths import cyclic_topological_sort
 # TODO: move this back to the bot file as soon as the source issue is sorted
 # https://travis-ci.org/regro/00-find-feedstocks/jobs/388387895#L1870
 from .migrators import *
-$MIGRATORS = [Version(),
-              Compiler(),
-              #Noarch(),
-             ]
+$MIGRATORS = {Version(): 5,
```

Comment:
It would be nice if there was a comment above this line that says that value is the PR limit.  A better model would be if the instances took a PR limit, ie `Version(pr_limit=5)` and this returned to a list.

Reply:
Do you want the migrators to do the PRing themselves? Currently the migrators don't have any PRing functionality.

Reply:
I guess we could request the attribute during the loop.

Reply:
Yeah, that was the idea.  The migrators can store the data about how they are to be used.